src/labels/CustomLabels.labels

```diff
@@ -2207,6 +2207,14 @@
         <shortDescription>Message displayed indicating data is not available when enhanced RDs are enabled</shortDescription>
         <value>This page is not available in Enhanced Recurring Donations. Refresh your browser and return to NPSP Settings.</value>
     </labels>
+    <labels>
+        <fullName>RD2_PauseCannotExistOnScheduleChange</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Cannot change specific fields causing schedule change when a pause exists</shortDescription>
+        <value>You can&apos;&apos;t update the following field(s) because this Recurring Donation is paused: {0}. If you&apos;&apos;d like to update the field(s), you must deactivate the pause.</value>
```

Comment:
It looks like the single apostrophe is duplicated here, in two spots.

Reply:
It has to be due to `String.format`. If it was just one, it would get lost during the formatting.

Reply:
Oh. Very interesting. That feels like a bug, but sadly it's been this way for a really long time.
http://www.fishofprey.com/2011/03/apex-stringformat-syntax.html